[2024-07-15T23:59:41.811+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T23:59:41.816+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-07-15T23:59:41.821+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T23:59:41.823+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-07-15T23:59:41.824+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> from DB
[2024-07-15T23:59:41.831+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T23:59:41.834+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-07-15T23:59:41.835+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-07-15T23:59:41.836+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-07-15T23:59:41.836+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]>
[2024-07-15T23:59:41.836+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-07-15T23:59:41.838+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-07-15T23:59:41.840+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-07-15T23:59:41.840+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-07-15T23:59:41.840+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [queued]>
[2024-07-15T23:59:41.840+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-07-15T23:59:41.845+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): generate_dbt_docs> on 2024-07-15 23:54:38.113182+00:00
[2024-07-15T23:59:41.846+0000] {standard_task_runner.py:57} INFO - Started process 1685 to run task
[2024-07-15T23:59:41.848+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'run_dbt_model_crm_sales', 'generate_dbt_docs', 'manual__2024-07-15T23:54:38.113182+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_models_crm_sales.py', '--cfg-path', '/tmp/tmp9bqmvoen']
[2024-07-15T23:59:41.848+0000] {standard_task_runner.py:85} INFO - Job 134: Subtask generate_dbt_docs
[2024-07-15T23:59:41.849+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0xffff7ee0a550>]
[2024-07-15T23:59:41.868+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T23:59:41.869+0000] {task_command.py:410} INFO - Running <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [running]> on host f934d9af8cb8
[2024-07-15T23:59:41.869+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 1685)
[2024-07-15T23:59:41.869+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 1685)
[2024-07-15T23:59:41.870+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-07-15T23:59:41.870+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [running]> from DB
[2024-07-15T23:59:41.884+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-07-15T23:59:41.885+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-07-15T23:59:41.896+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-07-15T23:59:41.901+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_model_crm_sales' AIRFLOW_CTX_TASK_ID='generate_dbt_docs' AIRFLOW_CTX_EXECUTION_DATE='2024-07-15T23:54:38.113182+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-15T23:54:38.113182+00:00'
[2024-07-15T23:59:41.901+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-07-15T23:59:41.901+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-07-15T23:59:41.902+0000] {bash.py:186} DEBUG - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_model_crm_sales' AIRFLOW_CTX_TASK_ID='generate_dbt_docs' AIRFLOW_CTX_EXECUTION_DATE='2024-07-15T23:54:38.113182+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-15T23:54:38.113182+00:00'
[2024-07-15T23:59:41.902+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-07-15T23:59:41.902+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'dbt docs generate --profiles-dir /opt/airflow/dbt --project-dir /opt/airflow/dbt']
[2024-07-15T23:59:41.905+0000] {subprocess.py:86} INFO - Output:
[2024-07-15T23:59:42.814+0000] {subprocess.py:93} INFO - [0m23:59:42  Running with dbt=1.8.3
[2024-07-15T23:59:42.924+0000] {subprocess.py:93} INFO - [0m23:59:42  Registered adapter: ***=1.8.2
[2024-07-15T23:59:43.009+0000] {subprocess.py:93} INFO - [0m23:59:43  Unable to do partial parsing because config vars, config profile, or config target have changed
[2024-07-15T23:59:43.010+0000] {subprocess.py:93} INFO - [0m23:59:43  Unable to do partial parsing because profile has changed
[2024-07-15T23:59:43.746+0000] {subprocess.py:93} INFO - [0m23:59:43  Found 6 models, 4 sources, 652 macros
[2024-07-15T23:59:43.747+0000] {subprocess.py:93} INFO - [0m23:59:43
[2024-07-15T23:59:43.783+0000] {subprocess.py:93} INFO - [0m23:59:43  Concurrency: 4 threads (target='dev')
[2024-07-15T23:59:43.783+0000] {subprocess.py:93} INFO - [0m23:59:43
[2024-07-15T23:59:43.875+0000] {subprocess.py:93} INFO - [0m23:59:43  Building catalog
[2024-07-15T23:59:43.919+0000] {subprocess.py:93} INFO - [0m23:59:43  Catalog written to /opt/airflow/dbt/target/catalog.json
[2024-07-15T23:59:44.475+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-15T23:59:44.485+0000] {__init__.py:75} DEBUG - Lineage called with inlets: [], outlets: []
[2024-07-15T23:59:44.486+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_model_crm_sales.generate_dbt_docs manual__2024-07-15T23:54:38.113182+00:00 [running]> from DB
[2024-07-15T23:59:44.490+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-07-15T23:59:44.490+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=run_dbt_model_crm_sales, task_id=generate_dbt_docs, execution_date=20240715T235438, start_date=20240715T235941, end_date=20240715T235944
[2024-07-15T23:59:44.491+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 2.654506
[2024-07-15T23:59:44.496+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-07-15T23:59:44.508+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-07-15T23:59:44.527+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun run_dbt_model_crm_sales @ 2024-07-15 23:54:38.113182+00:00: manual__2024-07-15T23:54:38.113182+00:00, state:running, queued_at: 2024-07-15 23:54:38.125855+00:00. externally triggered: True>: 0 task(s)
[2024-07-15T23:59:44.527+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
